src/ResourceManager/EventHub/Commands.EventHub.Test/ScenarioTests/Common.ps1

```diff
@@ -1,72 +0,0 @@
-﻿# ----------------------------------------------------------------------------------
-#
-# Copyright Microsoft Corporation
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-# http://www.apache.org/licenses/LICENSE-2.0
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-# ----------------------------------------------------------------------------------
-
-
-<#
-.SYNOPSIS
-Gets a resource group name for testing.
-#>
-function Get-ResourceGroupName
-{
-    return getAssetName
-}
-
-
-<#
-.SYNOPSIS
-Gets the location for the Website. Default to West US if none found.
-#>
-function Get-Location
```

Comment:
If these two functions were working can you add them back in and use them in place of Get-Location? I will continue looking at the issue you were seeing with Get-Location.